serve/filters/filter/tx.go

```diff
@@ -0,0 +1,101 @@
+package filter
+
+import (
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+)
+
+type Options struct {
+	Height    int64
+	Index     uint32
```

Comment:
When would we need to use these filter params?
I think we can safely drop them, this information is contained within the transaction result that's returned to the end user